src/seedu/addressbook/AddressBook.java

```diff
@@ -1158,12 +1158,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replace(prefix, "");
```

Comment:
There are a couple of things you need to take care with this commit.
1. Notice that you haven't updated the method to remove the prefix "the correct way"; your solution is still the same as the given approach. Please read the description carefully.
2. When you are abstracting the methods to make them generic, the header comment should reflect the same; your header comment is still specific to address book due to reference to p/ d/ etc.,